src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/scope_map.py

```diff
@@ -0,0 +1,145 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.util import CLIError
+from ._utils import get_resource_group_name_by_registry_name
+
+
+def _parse_actions_from_repositories(allow_or_remove_repository):
+    actions = []
+
+    allow_or_remove_repository.sort()
+    for rule in allow_or_remove_repository:
+        splitted = rule.split(';', 1)
+        if len(splitted) != 2:
+            return False, rule
+        repository, repository_actions = splitted[0], splitted[1].split(',')
+        for action in repository_actions:
+            actions.append("repositories/" + repository + "/" + action)
+
+    return True, actions
+
+
+def acr_scope_map_create(cmd,
+                         client,
+                         registry_name,
+                         scope_map_name,
+                         add_repository,
+                         resource_group_name=None,
+                         description=None):
+
+    validated, actions = _parse_actions_from_repositories(add_repository)
+    if not validated:
+        raise CLIError("Rule {} has invalid syntax.".format(actions))
+
+    resource_group_name = get_resource_group_name_by_registry_name(cmd.cli_ctx, registry_name, resource_group_name)
```

Comment:
If scope-maps is only available for `Premium` sku, please use `validate_premium_registry` func from _utils intead. Otherwise, ignore.

Reply:
I think it's available for all SKUs, but I will confirm it.